selene/support/shared/config.py

```diff
@@ -386,11 +386,11 @@ def type_by_js(self, value: bool):
         self._type_by_js = value
 
     @Config.window_width.setter
-    def window_width(self, value: Optional[int]):
+    def window_width(self, value: int | None):
         self._window_width = value
 
     @Config.window_height.setter
-    def window_height(self, value: Optional[int]):
+    def window_height(self, value: int | None):
```

Comment:
This is a python 3.10 feature https://www.python.org/dev/peps/pep-0604 
It shouldn't be here. The task is to up to 3.7 python, not 3.10.